input/europe/template2.xml.in

```diff
@@ -0,0 +1,6 @@
+        <!-- {{country}} -->
+        <entry>
+          <prototype>{{ reactor_name }}</prototype>
+          <number>1</number>
+        </entry>
+      
```

Comment:
may want to be careful about extra newlines. These are allowed in xml in most places, but the final file will be easier to handle without them.